hack/e2e-node-test.sh

```diff
@@ -129,6 +129,6 @@ else
   # Test using the host the script was run on
   # Provided for backwards compatibility
   "${ginkgo}" --focus=$focus --skip=$skip "${KUBE_ROOT}/test/e2e_node/" --report-dir=${report} \
-    -- --alsologtostderr --v 2 --node-name $(hostname) --build-services=true --start-services=true --stop-services=true
+    -- --alsologtostderr --v 2 --node-name $(hostname) --disable-kubenet=true --build-services=true --start-services=true --stop-services=true
```

Comment:
This is just for local node e2e test #28516 -> `make test_e2e_node`.
For local development, it is a little complicated to download cni and nsenter, because the developer may have different environment. At least the nsenter installation script doesn't work on my desktop. :(